.github/ISSUE_TEMPLATE/stabilization.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Module stabilization
+about: Stabilize a module before a 1.0 release
+title: 'Stabilize module X'
+labels: 'stabilization'
+assignees: ''
+---
+
+Before stabilizing a module, an approver or maintainer must make sure that the following criteria are met:
+
+- [ ] One RC release or more have been done of this module
+- [ ] No open issues or PRs in the module that would require breaking changes
+- [ ] No TODOs in the module code that would require breaking changes
+- [ ] No deprecated symbols in the module
+- [ ] No symbols marked as experimental in the module
```

Comment:
Curious why? Seems common for stable module to have experimental APIs, e.g. grpc-go.

Reply:
We discussed a bit about this on #4705 and #4832. Personally, I think grpc-go's approach has caused a lot of pain to end-users; I gathered some issues related to it on https://github.com/open-telemetry/opentelemetry-collector/issues/4832#issuecomment-1039394229.

We also have the experience of opentelemetry-go, where their decision to have interfaces that can change across minor versions has caused some pain (see open-telemetry/opentelemetry-go/issues/3277)

We can discuss this more explicitly (I think having a more explicit guideline as to how to implement experimental methods is important), but for now I think requiring the 1.0 version to not have experimental symbols is not too onerous

Reply:
Thanks for the context, I agree the implication of experimental APIs can be a separate discussion. Makes sense to me to not have them for 1.0 to begin with (since relaxing this constraint in the future is feasible).